pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -1701,6 +1716,14 @@ func isImmutable(cr *api.KieApp) bool {
 	return false
 }
 
+func isTrial(cr *api.KieApp) bool {
```

Comment:
What is this trial mode?

Reply:
The Trial mode is the most simplier combination we can have when deploying RHPAM/DM.
It deploys the KIE Server and the Business/Decision Central with no PV and database used is embedded, the H2.
plus, it uses http routes.

Reply:
thanks for explaining it.